docs/installation.rst

```diff
@@ -18,8 +18,18 @@ Install from PyPi
 
 .. code-block:: shell
 
-   $ pip install ansible-builder
+   $ pip3 install ansible-builder
 
+.. note::
+
+   An **alternative** approach to install ``ansible-builder`` is using the ``ansible-dev-tools`` package.
```

Comment:
Do we really want to put "alternative" in bold text? This is already in a note to catch user attention. I'm not sure what emphasizing "alternative" is meant to convey here tbh.